interfaces/builtin/maliit.go

```diff
@@ -106,16 +109,19 @@ dbus (send)
     bus=session
     interface="org.maliit.Server.Address"
     path=/org/maliit/server/address
-    peer=(label=###SLOT_SECURITY_TAGS###),
+    peer=(label=unconfined),
```

Comment:
Hmm, can you explain why this is being changed? The slot may be confined and even if it is not, it should have been handled in the method logic, not in the snippet IMO.